server/rpm/pbench-server.spec.j2

```diff
@@ -8,30 +8,11 @@ URL:            http://github.com/distributed-systems-analysis/pbench
 Source0:        pbench-server-%{version}.tar.gz
 Buildarch:      noarch
 
-
-%if 0%{?rhel} == 7
-# need python3 - install scl to get it
-# scl-utils is in the main repo, but the scl python3
-# collection du jour is in the scl repo which
-# is assumed to have been added already with:
-#    yum install yum-utils
-#    yum-config-manager --enable rhel-server-rhscl-7-rpms
-#
-# we can't do that in the spec file, because of the yum lock,
-# so they have to be done beforehand.
-Requires: scl-utils, rh-python36
-%else
-Requires: python3 python3-devel gcc
-%endif
+Requires: python38 python38-six python38-urllib3
```

Comment:
Why is this completely different from before? Assuming we don't actually need `python3-devel` or `gcc`, where does the 0.69 server require `six` or `urllib3`??

I assume these are changes you needed to make to get an install last night, but it'd be nice to capture some of the history behind this.

Reply:
> Why is this completely different from before?

Because before we used the SCL environment to gain Python3 on RHEL 7 boxes which did not provide it natively.  We may have needed to have `python3-devel` and `gcc` present in the past in order to install certain "pip" installed modules, but it is not clear if that is really the case with SCL.

Now that we are focusing the pbench production server installs only on RHEL 8, we move to Python 3.8.

> Assuming we don't actually need `python3-devel` or `gcc`

There is no assumption here.  I have verified and tested the installation on a RHEL 8 box with Python 3.8 showing that we do not need either of those two RPMs.  That is to say our production environment is running just fine without these two packages installed.

>, where does the 0.69 server require `six` or `urllib3`??

They are implicit requirements that come from the Python3 modules listed in our `requirements.txt` file.  We add these explicit dependencies here so that the local `pip3` install mechanism does not have to install them, finding them already present in the global installation of Python 3.8 `site-packages`.

> I assume these are changes you needed to make to get an install last night, but it'd be nice to capture some of the history behind this.

I can add a comment describing why we add these RPM dependencies.